app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -147,8 +147,7 @@ class NewMessageFragment : Fragment() {
 
         setWebViewReference()
         initUi()
-
-        if (newMessageViewModel.initResult.value == null) initDraftAndViewModel()
+        initDraftAndViewModel()
```

Comment:
Why do we need to recreate the draft every time we come back here? The view model should be able to keep the data in memory and we could directly start setting up the view